tests/ops/qubit/test_attributes.py

```diff
@@ -451,3 +452,22 @@ def test_qaoa_embedding(self, features, weights, num_wires, batch_size):
             features, weights, wires=list(range(num_wires)), local_field=qml.RY
         )
         op.decomposition()
+
```

Comment:
Please consider adding a test that applies the PCP gate to a simple circuit and returns the state.

Reply:
Added 👍🏼